tools/oeutil/enc/generate_evidence_enc.cpp

```diff
@@ -134,15 +134,15 @@ oe_result_t get_plugin_evidence(
 
     OE_CHECK(oe_get_evidence(
         &evidence_format,
-        OE_EVIDENCE_FLAGS_EMBED_FORMAT_ID,
+        0,
         NULL,
         0,
         NULL,
         0,
         &local_evidence,
         &local_evidence_size,
-        endorsements ? &local_endorsements : NULL,
-        endorsements ? &local_endorsements_size : 0));
+        &local_endorsements,
```

Comment:
These 2 lines are intentional. Passing in non-null endorsement unnecessarily make calls to retrieve endorsements which can be time consuming. Please restore.